.docker/nginx/sites/opendor.me.conf

```
@@ -0,0 +1,29 @@
+upstream php-upstream {
+   server app:9000;
+}
+
+server {
+    listen 80;
+
+    index index.php index.html;
```

We will never use a HTML entry file here
```suggestion
    index index.php;
```